src/Env.js

```diff
@@ -1,5 +1,3 @@
-var MIDDLE_TIER_URL = (process.env.NODE_ENV === 'development' ?
-        "http://localhost:8080" :
-        "https://cs4500-sp19-random1.herokuapp.com")
+var MIDDLE_TIER_URL = "https://cs4500-sp19-random1.herokuapp.com"
```

Comment:
I'm not sure why this is needed / if this work as is because I can't check it out and test it. What errors were you seeing on prod? I didn't notice any issues with the FAQ data being pulled in.

Reply:
Our app always thinks we are in development, so it always returns localhost. I confirmed this - that's why I made the change. I also pushed it to heroku, that's why you're not seeing any issues now, but before I pushed it this problem always existed.

Reply:
I'm confused, how did you push the change to Heroku before it got onto master?

Reply:
Please tell me you haven't been deploying development code to our heroku